tests/codegen/fcode/scripts/classes_3.py

```diff
@@ -9,7 +9,7 @@ def __del__(self : 'Point'):
         pass
 
     def translate(self : 'Point', a : 'float[:]'):
-        self.x = self.x + a
+        self.x[:] = self.x + a
```

Comment:
Yes this works the same. I have just noticed that the only difference between this file and `tests/codegen/fcode/scripts/classes_2.py` is that this file calls the constructor by passing a list instead of an array. As lists are added and more strictly typed this should break. Should I remove this file?